src/modules/ens/ens.service.ts

```diff
@@ -181,15 +179,15 @@ export class EnsService {
     owner,
   }: {
     hash: string;
-    name: string;
+    name?: string;
     owner?: string;
   }) {
     try {
       const namespaceOwner = owner ? owner : await this.ensRegistry.owner(hash);
-    
+
       if (namespaceOwner === emptyAddress) {
         //prevent resync and remove namespace from database
-        return this.deleteNamespace(name);
+        return this.deleteNamespace(hash);
       }
    
       const data = await this.domainReader.read({ node: hash });
```

Comment:
Does `domainReader.read()` read return the `name`? If not, maybe we can use `domainReader.readName()` here as we now know that the namespace isn't deleted?